stumpy/stumped.py

```diff
@@ -124,13 +131,27 @@ def stumped(dask_client, T_A, m, T_B=None, ignore_trivial=True):
     M_T, Σ_T = core.compute_mean_std(T_A, m)
     μ_Q, σ_Q = core.compute_mean_std(T_B, m)
 
+    T_A[np.isnan(T_A)] = 0
+
     out = np.empty((l, 4), dtype=object)
     profile = np.empty((l,), dtype="float64")
     indices = np.empty((l, 3), dtype="int64")
 
     hosts = list(dask_client.ncores().keys())
     nworkers = len(hosts)
 
+    step = 1 + l // nworkers
+    QT_futures = []
+    QT_first_futures = []
+    for i, start in enumerate(range(0, l, step)):
+        profile[start], indices[start, :] = _get_first_stump_profile(
+            start, T_A, T_B, m, excl_zone, M_T, Σ_T, ignore_trivial
+        )
+
+    T_B[
+        np.isnan(T_B)
+    ] = 0  # After having calculated all first profiles, we can remove nans from T_B
```

Comment:
Replace comment with `  # Remove all nan values from T_B only after first profile is calculated`